### ISSUE TITLE
Duplicate workflow discovery when running in `~/.molflow/workflows`

### ISSUE THREAD

**Issue by avirshup** [molecular-workflow-repository/molflow#2]
Everything runs normally everywhere else.

```bash
[~/.molflow/workflows]$ molflow list
A workflow named 'add_two_to_it' was found via multiple local workflow paths.
Locations found: [[PosixPath('~/.molflow/workflows'), PosixPath('.')], [PosixPath('~/.molflow/workflows'), PosixPath('.')]]
A workflow named 'convert' was found via multiple local workflow paths.
Locations found: [[PosixPath('~/.molflow/workflows'), PosixPath('.')], [PosixPath('~/.molflow/workflows'), PosixPath('.')]]
A workflow named 'count_atoms' was found via multiple local workflow paths.
Locations found: [[PosixPath('~/.molflow/workflows'), PosixPath('.')], [PosixPath('~/.molflow/workflows'), PosixPath('.')]]
A workflow named 'parameterize_small_molecule' was found via multiple local workflow paths.
Locations found: [[PosixPath('~/.molflow/workflows'), PosixPath('.')], [PosixPath('~/.molflow/workflows'), PosixPath('.')]]
Workflow location: Current directory
 - add_two_to_it (v0.1.0): Add 2 to integer, float, or complex number.
 - convert (v0.1.0): Translate a molecule between a variety of file formats, chemical ids, and python objects.
 - count_atoms (v0.0.1): Returns the number of atoms in a molecule.
 - parameterize_small_molecule (v0.0.1): Automatically parameterize a small molecule.
 - add_two_to_it (v0.1.0): Add 2 to integer, float, or complex number.
 - convert (v0.1.0): Translate a molecule between a variety of file formats, chemical ids, and python objects.
 - count_atoms (v0.0.1): Returns the number of atoms in a molecule.
 - parameterize_small_molecule (v0.0.1): Automatically parameterize a small molecule.

Workflow location: Current directory
 - add_two_to_it (v0.1.0): Add 2 to integer, float, or complex number.
 - convert (v0.1.0): Translate a molecule between a variety of file formats, chemical ids, and python objects.
 - count_atoms (v0.0.1): Returns the number of atoms in a molecule.
 - parameterize_small_molecule (v0.0.1): Automatically parameterize a small molecule.
 - add_two_to_it (v0.1.0): Add 2 to integer, float, or complex number.
 - convert (v0.1.0): Translate a molecule between a variety of file formats, chemical ids, and python objects.
 - count_atoms (v0.0.1): Returns the number of atoms in a molecule.
 - parameterize_small_molecule (v0.0.1): Automatically parameterize a small molecule.
```

**Comment by joseph-schaeffer** [comment]
This is technically intended, but we should probably do some sort of special case for it. The reason it happens is the default config includes both `./` and `~/.molflow/workflows/` as local repo locations. I can add a check that looks to see if those happen to be the same thing, and if so, only includes one of them.

**Comment by joseph-schaeffer** [comment]
Or rather, more generally check to make sure we eliminate duplicates from the config local repo paths.

**Comment by joseph-schaeffer** [comment]
Fixed by PR #4.
